### PR TITLE
shrshinde/isSupported-menus-check

### PR DESCRIPTION
**Pull request by shrshindeMSFT**
- Adding isSupported check to menus capability
- Adding unit test cases for menus capability

### CODE REVIEW

File: packages/teams-js/test/public/menus.spec.ts [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+      };
+
+      it('should not allow calls before initialization', () => {
+        expect(() => menus.setUpViews([viewConfiguration], () => true)).toThrowError(
+          'The library has not yet been initialized',
+        );
+      });
+
+      Object.values(FrameContexts).forEach(frameContext => {
+        it(`menus.setUpViews should throw error when menus is not supported  when set to true in ${frameContext} context`, async () => {
```
Comment (erinha) [inline packages/teams-js/test/public/menus.spec.ts:39]: nit: extra space between "not supported" and "when set"

Comment (erinha) [reply packages/teams-js/test/public/menus.spec.ts:39]: Also I think this was copy/pasted elsewhere. :)

**Review by erinha — APPROVED** [review]
Nit of a comment. Looks good otherwise, thanks for adding unit tests!

**Review by aeunms — APPROVED** [review]
+1 to Erin's nit comments, but it's still approved!

**Pull request merged**